config.proto

```diff
@@ -31,7 +30,7 @@ message Reporting {
     // address represents the host for reporting the traces e.g. api.traceable.ai
     google.protobuf.StringValue address = 1;
 
-    // secure when false, permits connecting to the trace endpoint without a certificate
+    // when `true`, avoids connecting to the trace endpoint without a certificate
```

Comment:
this comment is a little hard to read.  maybe:
when `true`, connects to endpoints over tls.

Reply:
Done.

Reply:
Is this getting reflected in `ENV_VARS.md`?

Reply:
Done.